.github/main.workflow

```diff
@@ -0,0 +1,16 @@
+workflow "Publish Helm chart" {
+  on = "push"
+  resolves = ["helm-gh-pages"]
```

Comment:
Can the action not be gated on the tag, here?

Reply:
Sorry I don't understand your question, can you please elaborate? The `tag` gate is inside the `helm-gh-pages` action. The linter will run on every push.

Reply:
What happens when the linter fails?

Reply:
The workflow stops and errors out, marking the PR as invalid or stopping the publishing if we're dealing with git tag.